.env.example

```diff
@@ -0,0 +1,3 @@
+GOOGLE_OAUTH_URL=http://api.dev.commonground.tw/api/oauth2/google
+
+NEXT_PUBLIC_TEST_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjEyMzQ1Njc4OTAiLCJ1c2VybmFtZSI6IkNvbW1vbmdyb3VuZCIsIm5pY2tuYW1lIjoiY29tbWluR3JvdW5kIiwiZW1haWwiOiJqd3RAY29tbW9uZ3JvdW5kLnR3IiwiYXZhdGFyIjoiaHR0cHM6Ly9pbWd1ci5jb20va1F2WHVrTi5wbmciLCJleHAiOjE3MzQxMjkyNDF9.SGJG2rxAJ97PO_cI67XanQflAJ5Ry7fffrsYBiYvsOw
```

Comment:
Consider changing this to YOUR_TOKEN_HERE instead of providing a token (which lacks correct fields).